### ISSUE TITLE
Remove hardcoded color used for the button hover state 

### ISSUE THREAD

**Issue by MisRob** [learningequality/kolibri#12553]
🌱 **Are you new to the codebase? Welcome! Please see [the contributing guidelines](https://github.com/learningequality/kolibri/blob/develop/CONTRIBUTING.md).**

## Blocked by

- https://github.com/learningequality/kolibri-design-system/issues/726

## Summary

After Kolibri Design System [provides `$darken_` utilities](https://github.com/learningequality/kolibri-design-system/issues/726), let's use them to get rid of this hardcoded color https://github.com/learningequality/kolibri/blob/7ba25fd14e2df6127879111922316f1225a6d787/kolibri/plugins/coach/assets/src/views/common/QuizStatus.vue#L350 by applying the most appropriate `$darken_` on the base red color color that the button has when it's not in hovered state.

**Comment by lokesh-sagi125** [comment]
hey , can i get assigned to this issue?

**Comment by MisRob** [comment]
Hi @lokesh-sagi125, welcome! Thanks for volunteering, we appreciate it.

Before this issue can be worked on, https://github.com/learningequality/kolibri-design-system/issues/726 needs to be resolved, and another contributor is already assigned to https://github.com/learningequality/kolibri-design-system/issues/726.

We welcome you on another issue! There are contributing opportunities in three repositories. See the contributing guidelines including links to issues suitable for contribution for each repository here:

- [Kolibri Contributing Guidelines](https://github.com/learningequality/kolibri/blob/release-v0.16.x/CONTRIBUTING.md)
- [Kolibri Design System Contributing Guidelines](https://github.com/learningequality/kolibri-design-system/blob/main/CONTRIBUTING.md)
- [Studio Contributing Guidelines](https://github.com/learningequality/studio/blob/unstable/CONTRIBUTING.md)

You can also see the list of unassigned ["help wanted"](https://github.com/search?q=org%3Alearningequality+is%3Aopen+is%3Aissue+label%3A%22help+wanted%22+no%3Aassignee&type=issues) or ["good first issue"](https://github.com/search?q=org%3Alearningequality+is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22+no%3Aassignee&type=issues) issues across all repositories.

**Comment by shivam-daksh** [comment]
Hey @MisRob , If I would work on this issue.

**Comment by MisRob** [comment]
Hi again @shivam-daksh , assigned, thanks a lot. 

In Kolibri, you can [use `devserver-with-kds` command](https://kolibri-dev.readthedocs.io/en/develop/howtos/development_with_kds.html) to run Kolibri with your local KDS so you can access your work in https://github.com/learningequality/kolibri-design-system/pull/728.

**Comment by Aditya8840** [comment]
Hi @shivam-daksh 
I wanted to check if you’re currently working on this issue. If not, I would love to contribute and take it up.

**Comment by shivam-daksh** [comment]
Hi @Aditya8840, An issue in [#4634](https://github.com/learningequality/studio/issues/4634) was persisting. That's the reason it took a long delay. I'll work on this issue.

**Comment by MisRob** [comment]
Thanks @shivam-daksh and welcome @Aditya8840.

@Aditya8840, there are other contributing opportunities in three repositories. See the contributing guidelines including links to issues suitable for contribution for each repository here:

- [Kolibri Contributing Guidelines](https://github.com/learningequality/kolibri/blob/release-v0.16.x/CONTRIBUTING.md)
- [Kolibri Design System Contributing Guidelines](https://github.com/learningequality/kolibri-design-system/blob/main/CONTRIBUTING.md)
- [Studio Contributing Guidelines](https://github.com/learningequality/studio/blob/unstable/CONTRIBUTING.md)

You can also see the list of unassigned ["help wanted"](https://github.com/search?q=org%3Alearningequality+is%3Aopen+is%3Aissue+label%3A%22help+wanted%22+no%3Aassignee&type=issues) or ["good first issue"](https://github.com/search?q=org%3Alearningequality+is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22+no%3Aassignee&type=issues) issues across all repositories.

**Comment by shivam-daksh** [comment]
Hi @MisRob , I'm getting error while installing dependencies using `yarn install`. I've attached the log file.
[logs.txt](https://github.com/user-attachments/files/16835671/logs.txt)

**Comment by akolson** [comment]
Hi @shivam-daksh! It appears, from the above, that you encountered some network connectivity issues(or the connection was not stable during the download) while downloading the dependencies. Could you delete the node_modules files and try to reinstall the dependencies again (`yarn install`)?
Below is a snippet from the attached logs.txt file attached
```
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
```

**Comment by shivam-daksh** [comment]
Hi @MisRob and @akolson , the issue is still persisting. I think it is not because of network error. Please check the logs in the end. I've tried with `yarn install` after deleting node_modules, still faced the same issue.

**Comment by MisRob** [comment]
Hi @shivam-daksh, I'm sorry to hear it's troubling.

I wanted to recommend you Volta to manage your node version, however it seems you are already using it, am I right? The node version looks correct.

Did you follow https://kolibri-dev.readthedocs.io/en/develop/getting_started.html closely step by step before running `yarn install`? To understand whether there was an issue even before running `yarn install`.

If you haven't done so already, I'd recommend to delete the entire folder and try one more time. If it doesn't help, would you message us at contributors@learningequality.org to receive Slack invitation and upload as many logs as possible from the whole setup process there?

**Comment by shivam-daksh** [comment]
Hi @MisRob,

I'm using Volta to manage my Node versions and followed the instructions from [the Kolibri documentation](https://kolibri-dev.readthedocs.io/en/develop/getting_started.html) exactly. When I tried this on GitHub Codespaces, the `yarn install` command completed without errors. However, after running `yarn run devserver` (or `devserver-hot`), the build was successful, but when I accessed the project through the forwarded port, I only saw Kolibri's loading screen with the logo. I waited for a while, but Kolibri never fully loaded.

I also tried deleting the entire folder and re-cloning the project on my local system, but I'm encountering the same issue with yarn install.

Any suggestions on how to resolve this?

**Comment by MisRob** [comment]
Looking at logs, this seems to be something similar? https://github.com/nodejs/node-gyp/issues/2681. Would you try play around with some guidance offered there if so?

**Comment by MisRob** [comment]
@shivam-daksh Looking at logs, this seems to be something similar? https://github.com/nodejs/node-gyp/issues/2681. Would you try play around with some guidance offered there if so?

**Comment by MisRob** [comment]
Closed by https://github.com/learningequality/kolibri/pull/12645
